firmware/release-notes.md

```diff
@@ -1,5 +1,9 @@
 # Raspberry Pi4 bootloader EEPROM release notes
 
+## 2020-06-17 Promote 2020-06-16 to STABLE
```

Comment:
small nit: this should probably be "Promote 2020-06-**15**" (not 16) to stable

Reply:
Fixed. thanks